src/main/java/com/sandvoxel/generitech/common/tileentities/machines/TileEntityPulverizer.java

```diff
@@ -276,20 +303,11 @@ public void update() {
             }
 
 
-            if (fuelRemaining == 0 && machineActive && machineTier == machineTier.TIER_0) {
-                ticksRemaining = 0;
-                test = false;
+            if (ticksRemaining <= 0 && machineActive)
+            {
 
-            } else {
-                test = false;
-            }
 
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS134)